peps/pep-0661.rst

```diff
@@ -337,27 +338,6 @@ idiom were unpopular, with the highest-voted option being voted for by only
 25% of the voters.
 
 
-Specific type signatures for each sentinel value
-------------------------------------------------
-
-For a long time, the author of this PEP strove to have type signatures for
-such sentinels that were specific to each value.  A leading proposal
-(supported by Guido and others) was to expand the use of ``Literal``, e.g.
-``Literal[MISSING]``.  After much thought and discussion, especially on the
-typing-sig mailing list [8]_, it seems that all such solutions would require
```

Comment:
To fix the CI error, either delete the corresponding footnote [8] at the bottom of the file, or if you want to keep it, turn it into a bullet point below.

Reply:
That's helpful, thank you!